Makefile.PL

```diff
@@ -39,8 +39,9 @@ requires
   'Locale::TextDomain'          => 1.20,
   ;
 
-test_requires 'DBD::SQLite';
+test_requires 'DBD::SQLite' => 1.4702;
 test_requires 'Test::NoWarnings';
+test_requires 'Time::Local' => 1.26;
```

Comment:
Time::Local is part of the Perl instllation, at lease on FreeBSD. Should we then really specify it again?

Reply:
It's one of [the standard modules](https://perldoc.perl.org/modules#Standard-Modules) so it should be included on all platforms. If we have a certain version requirement we do want to list it, but probably not otherwise.

Reply:
If so I think it is better to specify the Perl version required. That is easier for the users to check, and gives the same result.

Reply:
We require version 1.26 (use of the `timelocal_modern` function), and on CentOS 7 version 1.23 is installed.

Reply:
If Time::Local is available on CPAN it is fine.